lib/models/Secret.js

```diff
@@ -0,0 +1,46 @@
+const pool = require('../utils/pool');
+
+class Secret {
+  id;
+  title;
+  description;
+  createdAt;
+
+  constructor(row) {
+    this.id = row.id;
+    this.title = row.title;
+    this.description = row.description;
+    this.createdAt = row.createdAt;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query(
+      ` SELECT * 
+    FROM secrets`
+    );
+    return rows.map((row) => new Secret(row));
+  }
+  static async insert({ title, description }) {
+    const { rows } = await pool.query(
+      `INSERT into secrets (title, description) 
```

Comment:
if you wanted to use authorize middleware to only allow users to see secrets that they are associated with, you'd have to have some way to associate the data with a particular user, then you could check that they match in your authorize middleware